qiita_pet/handlers/api_proxy/prep_template.py

```diff
@@ -158,8 +158,8 @@ def prep_template_ajax_get_req(user_id, prep_id):
             download_qiime.append(fp_id)
         else:
             download_prep.append(fp_id)
-    download_prep = download_prep[0]
-    download_qiime = download_qiime[0]
+    download_prep = download_prep[0] if download_prep else None
```

Comment:
That is more concerning as that scenario should never happen. I don't think the solution is add the if statement here, but rather debug and find in which situation the QIIME map is not generated.